person.rb

```diff
@@ -1,25 +1,58 @@
 require_relative 'nameable'
+
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] to make this association work correctly Getters and Setters for @rentals should be present